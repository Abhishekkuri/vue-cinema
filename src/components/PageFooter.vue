<template>
  <div class="footer-container">
    <span>This is a Cinema application.</span>
    <span>Framework: VueJS</span>
    <span>Creator: Abhishek Kuri</span>
  </div>
</template>

<script>
export default {};
</script>

<style lang="scss" scoped>
.footer-container {
  background-color: $color-tirtiery;
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-evenly;
}
</style>
