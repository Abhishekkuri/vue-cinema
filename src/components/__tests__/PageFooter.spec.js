import { shallowMount } from "@vue/test-utils";
import PageFooter from "@/components/PageFooter";

describe("PageFooter.vue", () => {
  test("Footer renders correctly", () => {
    expect(shallowMount(PageFooter).element).toMatchSnapshot();
  });
});
