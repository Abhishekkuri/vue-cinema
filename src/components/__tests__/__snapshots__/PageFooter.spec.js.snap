// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter.vue Footer renders correctly 1`] = `
<div
  class="footer-container"
>
  <span>
    This is a Cinema application.
  </span>
   
  <span>
    Framework: VueJS
  </span>
   
  <span>
    Creator: Abhishek Kuri
  </span>
</div>
`;
